no-pdi/cmake/FindPDI.cmake

```diff
@@ -0,0 +1,45 @@
+include(CMakePrintHelpers)
+
+# Set the default search paths
+set(PDI_DIRS "${CMAKE_CURRENT_LIST_DIR}/..")
+
+# Search for a directory in CMAKE_MODULE_PATH that includes "no-pdi",
+# which is passed as an argument to the cmake command
+set(NO_PDI_PATH "")
+foreach(module_path IN LISTS CMAKE_MODULE_PATH)
```

Comment:
Why this search procedure? You know exactly where it is. It's your parent directory: `PDI_DIRS`